<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FanWidget</class>
 <widget class="QWidget" name="FanWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>32</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QLabel" name="nameLabel">
     <property name="text">
      <string>Core 0:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="valueLabel">
     <property name="text">
      <string>43C</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="highValueLabel">
     <property name="text">
      <string>high: 86C</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
